### PR TITLE
make pulpcore-selinux depend on correctly named pulpcore

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/pulpcore-selinux/pulpcore-selinux.spec [hunk 1]
```diff
@@ -15,8 +15,8 @@ BuildRequires:  checkpolicy
 BuildRequires:  selinux-policy-devel
 BuildRequires:  systemd
 Requires:       selinux-policy >= %{_selinux_policy_version}
-Requires:       python3-pulpcore
-Requires(post): policycoreutils, python3-pulpcore
+Requires:       %{?scl_prefix}python%{python3_pkgversion}-pulpcore
```
Comment (evgeni) [inline packages/pulpcore-selinux/pulpcore-selinux.spec:21]: scl_prefix doesn't seem to work…

Comment (evgeni) [reply packages/pulpcore-selinux/pulpcore-selinux.spec:21]: once you do it right, it works…

**Comment by evgeni** [comment]
it does what I wanted it to do, merging.

**Review by pcreech — COMMENTED** [review]
retro ACK

**Pull request merged**